### PR TITLE
Fix many miscellaneous spec problems

### PR DESCRIPTION
**Pull request by bvisness**
While doing my work for #67, I found many lingering issues with the memory64 spec. Hopefully this is the last of them. (I'm sure it's not.)

See the individual commits for descriptions of each change.

### CODE REVIEW

**Comment by bvisness** [comment]
I have no idea what's going on with the PDF build. I'll see if I can fix the issue, but I might need some assistance.

File: document/core/appendix/properties.rst [hunk 1]
```diff
@@ -526,49 +526,49 @@ where :math:`\val_1 \gg^+_S \val_2` denotes the transitive closure of the follow
...
 
 * The length of :math:`b^\ast` must equal :math:`\limits.\LMIN` multiplied by the :ref:`page size <page-size>` :math:`64\,\F{Ki}`.
 
-* Then the memory instance is valid with :ref:`memory type <syntax-memtype>` :math:`\limits`.
+* Then the memory instance is valid with :ref:`memory type <syntax-memtype>` :math:`\idxtype~\limits`.
 
 .. math::
    \frac{
-     \vdashmemtype \limits \ok
+     \vdashmemtype \idxtype~\limits \ok \\
```
Comment (rossberg) [inline document/core/appendix/properties.rst]: I think the misplaced `\\` here is what breaks the build. Just remove it.

**Review by rossberg — COMMENTED** [review]
LGTM modulo comment

File: document/core/appendix/properties.rst (hunk 1 above)
Comment (bvisness) [reply document/core/appendix/properties.rst]: Good catch, thanks!

**Comment by sbc100** [comment]
OK to land this now?

**Comment by bvisness** [comment]
Yep, should be good.

**Pull request merged**